L1Trigger/L1THGCal/plugins/concentrator/HGCalConcentratorProcessorSelection.cc

```diff
@@ -89,23 +109,16 @@ void HGCalConcentratorProcessorSelection::run(const edm::Handle<l1t::HGCalTrigge
         case superTriggerCellSelect:
           superTriggerCellImpl_->select(trigCellVecCoarsened, trigCellVecOutput);
           break;
-        case mixedBestChoiceSuperTriggerCell:
-          if (triggerTools_.isEm(module_trigcell.first)) {
-            bestChoiceImpl_->select(geometry_->getLinksInModule(module_trigcell.first),
-                                    geometry_->getModuleSize(module_trigcell.first),
-                                    trigCellVecCoarsened,
-                                    trigCellVecOutput);
-          } else {
-            superTriggerCellImpl_->select(trigCellVecCoarsened, trigCellVecOutput);
-          }
+        case coarseTriggerCellSelect:
+          trigCellVecOutput = trigCellVecCoarsened;
```

Comment:
This is a bit strange, should it be kept as a valid option, or should the user be warned?
From outside it's unclear what is the difference between the two coarsenings, `coarsenTriggerCells_` applies to the full detector, while the other coarsening can vary between subdetectors. Would it make sense to also have a `coarsenTriggerCells_` that depends on the subdet, and therefore `coarseTriggerCellSelect` would just be a "noSelection"?

Reply:
Yes - I have now done as you suggest and `coarsenTriggerCells_` is also a vector (I had to change to vector of ints, since vector of bools doesn't seem to be supported in the conf files?).